examples/medplum-photon-integration/src/bots/handle-prescription-event.ts

```diff
@@ -64,6 +67,32 @@ export async function handler(medplum: MedplumClient, event: BotEvent<PhotonWebh
   return prescription;
 }
 
+async function getMedicationCodeFromPrescription(
+  prescriptionId: string,
```

Comment:
nit: helps keep things straight when integrating
```suggestion
  photonPrescriptionId: string,
```